app/javascript/react/screens/App/Overview/screens/PlanWizard/components/PlanWizardScheduleStep/PlanWizardScheduleStep.js

```diff
@@ -4,42 +4,73 @@ import { Field, reduxForm } from 'redux-form';
 
-export const PlanWizardScheduleStep = ({ targetProvider, migration_plan_choice_radio }) => (
-  <Form className="form-horizontal">
+export const PlanWizardScheduleStep = ({ targetProvider, migration_plan_type_radio, migration_plan_choice_radio }) => (
+  <Form className="form-vertical">
     <Field
-      name="migration_plan_choice_radio"
-      label={__('Run migration plan')}
+      name="migration_plan_type_radio"
       component={FormField}
       type="radio"
-      labelWidth="3"
+      label={__('Migration Type')}
+      inline_label={false}
+      controlWidth={12}
       options={[
         {
-          name: __('Save migration plan to run later'),
-          id: 'migration_plan_later'
+          id: 'migration_type_cold',
+          name: __('Cold Migration - Source VMs are shut down while all of the VM data is migrated.')
         },
         {
-          name: __('Start migration immediately'),
-          id: 'migration_plan_now'
+          id: 'migration_type_warm',
+          name: __(
+            'Warm Migration - VM data is iteratively pre-copied. A final, cutover migration is scheduled and run later.'
+          )
         }
       ]}
-      help={
-        targetProvider === 'openstack' && migration_plan_choice_radio === 'migration_plan_now' ? (
-          <div className="plan-wizard-vm-power-warning">
-            <Icon type="pf" name="warning-triangle-o" />
-            <p>
-              {__('VMs must be powered on in order to migrate.')}
-              <br />
-              {__('Ensure that all VMs in the Migration Plan are powered on before selecting Create.')}
-            </p>
-          </div>
-        ) : null
-      }
     />
+    {migration_plan_type_radio === 'migration_type_cold' && (
+      <Field
+        name="migration_plan_choice_radio"
+        component={FormField}
+        type="radio"
+        label={__('Schedule')}
+        inline_label={false}
+        style={{ paddingTop: '100px' }}
+        options={[
+          {
+            name: __('Save migration plan to run later'),
+            id: 'migration_plan_later'
+          },
+          {
+            name: __('Start migration immediately'),
+            id: 'migration_plan_now'
+          }
+        ]}
+        help={
+          targetProvider === 'openstack' && migration_plan_choice_radio === 'migration_plan_now' ? (
+            <div className="plan-wizard-vm-power-warning">
+              <Icon type="pf" name="warning-triangle-o" />
+              <p>
+                {__('VMs must be powered on in order to migrate.')}
+                <br />
+                {__('Ensure that all VMs in the Migration Plan are powered on before selecting Create.')}
+              </p>
+            </div>
+          ) : null
+        }
+      />
+    )}
+    {migration_plan_type_radio === 'migration_type_warm' && (
+      <Form.HelpBlock style={{ paddingLeft: '22px', paddingTop: '100px' }}>
+        {__(
```

Comment:
Same here, `/* prettier-ignore */ right before the closing `}` is how I usually do it

Reply:
Fixed as well. Thanks Mike.